src/system_simulation/server/test_models/seesaw_3.json

```diff
@@ -7,6 +7,7 @@
       "x": 2948.4185869449298,
       "y": 1765.4563241484457,
       "z": 18.510048737482713,
+      "added_mass": 0,
```

Comment:
where does that "added_mass" come frome? Because I see no changes in the ```json_export.rb```. Aren't this files from the json_export?